### DISCUSSION TITLE
Supabase as backend for external APIs?

### DISCUSSION THREAD

**Discussion by CSumm** [supabase/supabase#3975] [Questions] [answered]
Hi All,

Im planning out a project that will use external APIs and want to use Supabase as database and backend. 

I wanted to know if APIs can be directly included into Supabase and then called from the front end. 

In other words, would I be able to use an API such as for Eventbrite and store the backend logic and oauth keys on Supabase, then query it from the client?

Thank you in advance!

**Comment by dshukertjr** [comment, marked as answer]
Hi @CSumm!

At the moment, you will have to write your backend logic somewhere else since Supabase does not have Functions capability currently (might change in the future). In the mean while, you can use services like [Vercel's functions](https://vercel.com/docs/concepts/functions/introduction) to host your backend logic to access Eventbrite API.

**Reply by CSumm** [reply]
Awesome thank you! I ended up finding another way of handling the logic, and using Supabase as a service for Gatsby front end, and just connecting to Eventbrite via Gatsby as well.
